src/utils/user-preference.ts

```diff
@@ -1,10 +1,10 @@
 export default class Preference {
-  static getCitationStyle(): string | null {
-    return Office.context.document.settings.get('Style') as string | null;
+  static getItem(key: string): string | null {
```

Comment:
I think its better if only the Preference class knows about the settings keys and how the data is (de)serialized. Thus, I liked the previous version better